Assets/Scripts/Managers/HuntingInputManager.cs

```diff
@@ -38,6 +47,25 @@ protected override void Awake()
 
 		// ADS
 		playerInput.Hunting.ADS.performed += ctx => EquipmentManager.Instance.EquippedItem.UseSecondary();
+
+		// Lure
+		//playerInput.Hunting.ThrowLure.performed += ctx => Lure();
+	}
+
+	private void Update()
+	{
+		//Lure throwing code
+		//TODO: bugfix on why the new input system doesn't work
+		if (lureReload == true && Input.GetKeyDown(KeyCode.G) )
```

Comment:
Will need this changed over to the new input system, you can ask Kai or myself if you need help getting this to work.